src/EnergyPlus/ChillerReformulatedEIR.cc

```diff
@@ -342,56 +355,77 @@ namespace ChillerReformulatedEIR {
 				ErrorsFound = true;
 			}
 
-			ElecReformEIRChiller( EIRChillerNum ).EIRFPLRName = cAlphaArgs( 4 );
-			ElecReformEIRChiller( EIRChillerNum ).ChillerEIRFPLR = GetCurveIndex( cAlphaArgs( 4 ) );
+			//The default type of part-load curve is: LeavingCondenserWaterTemperature
+			if ( lAlphaFieldBlanks( 4 ) ) {
+				PartLoadCurveType = "LeavingCondenserWaterTemperature";
+			}
+			else {
+				PartLoadCurveType = cAlphaArgs( 4 );
+			}
+			
+			ElecReformEIRChiller( EIRChillerNum ).EIRFPLRName = cAlphaArgs( 5 );
+			ElecReformEIRChiller( EIRChillerNum ).ChillerEIRFPLR = GetCurveIndex( cAlphaArgs( 5 ) );
 			if ( ElecReformEIRChiller( EIRChillerNum ).ChillerEIRFPLR == 0 ) {
 				ShowSevereError( RoutineName + cCurrentModuleObject + "=\"" + cAlphaArgs( 1 ) + "\"" );
-				ShowContinueError( "Invalid " + cAlphaFieldNames( 4 ) + '=' + cAlphaArgs( 4 ) );
+				ShowContinueError( "Invalid " + cAlphaFieldNames( 5 ) + '=' + cAlphaArgs( 5 ) );
 				ErrorsFound = true;
 			}
-
-			// Chilled water inlet/outlet node names are necessary
-			if ( lAlphaFieldBlanks( 5 ) ) {
+			
+			//Check the type of part-load curves implemented: 1_LeavingCondenserWaterTemperature, 2_Lift    zrp_Aug2014
+			if ( SameString( PartLoadCurveType, "LeavingCondenserWaterTemperature" ) && SameString( GetCurveType( ElecReformEIRChiller( EIRChillerNum ).ChillerEIRFPLR ), "BICUBIC" ) ) {
```

Comment:
@mjwitte 
It seems that the users are required to use "BICUBIC" , rather than other curve types, to describe this specific curve ("Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve"). Details can be found in the following iDD sections. Please let me know if you think "warning" is better. Of course, table types need to be added if we keep using "severe error"

Chiller:Electric:ReformulatedEIR,
  A5 , \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Name
       \type object-list
       \object-list BicubicCurves
       \object-list BiVariateTables
       \object-list ChillerPartLoadWithLiftCurves
       \note two types of part-load curves are available: BicubicCurves or ChillerPartLoadWithLiftCurves
       \note Type 1: BicubicCurves; 
       \note Calculated based on LCT and PLR       
       \note curve = a + b*LCT + c*LCT**2 + d*PLR + e*PLR**2 + f*LCT*PLR + g*0 + h*PLR**3
       \note + i*0 + j*0
       \note PLR = part load ratio (cooling load/steady state capacity)
       \note LCT = leaving condenser fluid temperature(C)
       \note Type 2: ChillerPartLoadWithLiftCurves 
       \note Calculated based on dT*, Tdev* and PLR
       \note curve = a + b*(dT*) + c*(dT*)**2 + d*PLR + e*PLR**2 + f*(dT*)*PLR + g*(dT*)**3
       \note + h*PLR**3 + i*(dT*)**2*PLR + j*(dT*)*PLR**2 + k*(dT*)**2*PLR**2 + l*(Tdev*)*PLR**3